charts/vmclarity/templates/yara-rule-server/service.yaml

```diff
@@ -0,0 +1,19 @@
+{{/*
+Headless Service does not include a ClusterIP so that the DNS record includes
+all Pod addresses. This makes it possible for the API Gateway to load balance
+all replicas instead of relying on kube-proxy to do it.
+*/}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "vmclarity.yaraRuleServer.name" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "vmclarity.yaraRuleServer.labels.standard" . | nindent 4 }}
+spec:
+  type: ClusterIP
+  ports:
+    - name: http
+      protocol: TCP
+      port: 9993
+      targetPort: 9993
```

Comment:
the default value of server_address is `":8080"`: https://github.com/openclarity/yara-rule-server/blob/acb3afa89a18707c8e163c4f43071946032eb062/pkg/config/config.go#L73